readme.md

```diff
@@ -17,10 +17,18 @@ Getting started
     ```
 
     And add the certificate to your keychain:
-
+    
+    - MacOS
     ```
     security import cert.pem; security add-trusted-cert cert.pem
     ```
+    
+    - Linux (Ubuntu, Debian)
+    ```
+    sudo cp cert.pem /usr/local/share/ca-certificates/
```

Comment:
I think you need to rename the file?

http://manpages.ubuntu.com/manpages/focal/man8/update-ca-certificates.8.html
`Certificates must have a  .crt
       extension in order to be included by update-ca-certificates.`

```suggestion
    sudo cp cert.pem /usr/local/share/ca-certificates/ft-google-amp.crt
```
Or something like this?

Reply:
agree and the name could be different because it's just concatenated to ca-certificates.crt along with the other certificates. Thanks for the remark, man!